fastlane/Fastfile

```diff
@@ -154,13 +157,16 @@ lane :update_hybrid_common_versions do |options|
 
   if dry_run
     UI.message("ℹ️  Nothing more to do, dry_run: true")
-  else
-    replace_text_in_files(
-      previous_text: previous_version_number,
-      new_text: new_version_number,
-      paths_of_files_to_update: files_to_update
-    )
+    next
   end
+
+  bump_phc_version_create_pr(
+    repo_name: "purchases-flutter",
+    files_to_update: files_to_update,
+    current_version: current_phc_version,
+    next_version: new_version_number,
+    open_pr: options[:open_pr]
+  )
```

Comment:
Should we add the `automatic_release` parameter?

Reply:
Oh good idea

Reply:
I added here, but not when calling it from `update_hybrid_common_versions_and_open_pr` because that one may not always be used for automatic releases. I added it to the CircleCI workflow tho